config/server/server.js

```diff
@@ -0,0 +1,19 @@
+const express = require('express');
+const { renderCallback, middlewares } = require('serverEntry').default;
+const port = process.env.PORT || 8080;
+
+const app = express();
+
+if (
+  middlewares &&
+  (middlewares instanceof Function ||
+    middlewares instanceof express ||
+    (middlewares instanceof Array && middlewares.length > 0))
```

Comment:
I don't really understand what the point of this conditional is.

```javascript

() => throw new Error('foo') // passes
[-1] // passes
[undefined] // passes

```

What sort of runtime failure are you trying to catch here?